### ISSUE TITLE
False positive for play links

### ISSUE THREAD

**Issue by guifelix** [sindresorhus/awesome-lint#122]
Having problems with play links as its reading as double-link

here is one example: ```https://play.google.com/store/apps/details?id=com.joaomgcd.autoweb```

**Comment by sindresorhus** [comment]
// @Scrum

**Comment by Scrum** [comment]
relate to #114
